src/Tickets/Admin/Provider.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
We're going to be [introducing this](https://github.com/the-events-calendar/event-tickets/pull/2228/files#diff-b1f0906773bc46a8f6daadde1516e93c2710ac28c92a66b97dc3aadf176ff9b4) as part of the tickets menu work, can you please check that it plays nicely with it?

Reply:
Looks like you placed it in the `Tribe` folder/namespace and not the new `Tickets` folder. Aren't we attempting to move away from `Tribe`? If so, then we'll definitely have to merge these two together when that time comes.

Reply:
Just to add to this: my `Provider` file utilizes a `Hooks` class instead of calling them directly from the `Provider` class. That's pretty much the biggest difference between the two. Otherwise, they're very compatible.